modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -29,8 +29,116 @@ payload
 
 // CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// Flat File config properties are shared with COBOL Copybook and Fixed Width.
-include::partial$dataweave-formats-flatfile-config.adoc[]
+
+[[properties]]
+== Configuration Properties
+
+DataWeave supports the following configuration properties for this format.
+
+
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
+
+
+[cols="1,1,1,3a", options="header"]
+|===
+|Parameter |Type |Default|Description
+|`allowLenientWithBinaryNotEndElement`  |`Boolean`|`false`|When the schema contains elements of type Binary or Packed, the `lenient` option does not allow short records, regardless of the last element's
```

Comment:
Reader or Writer property? 
Is `allowLenientWithBinaryNotEndElement` a reader or writer property. *The 2.4 doc shows it as a writer property.* See https://docs.mulesoft.com/dataweave/2.4/dataweave-formats-fixedwidth#properties

Reply:
added question to include

Reply:
Tomo confirmed that it is Reader property. 
Need to fix in the docs (all versions where it's listed).

Reply:
added to v2.4

Reply:
done in 2.5 of course